rust_src/src/threads.rs

```diff
@@ -33,4 +42,10 @@ pub fn thread_name(thread: LispObject) -> LispObject {
     thread.as_thread_or_error().name()
 }
 
+/// Return t if THREAD is alive, or nil if it has exited.
+#[lisp_fn]
+pub fn thread_alive_p(thread: LispObject) -> LispObject {
```

Comment:
This has to follow the elisp convention and is fine as is.

Reply:
Using the updated support in `lisp_fn` this can be simplified to:

    pub fn thread_alive_p(thread: ThreadStateRef) -> bool {
        thread.is_alive()
    }

You just need to add an `impl` for the `From` interface if there is not already one.

Reply:
Didn't realize `lisp_fn` had been changed as well, thanks for the heads-up!